civitai-downloader-v2/src/core/adaptability/plugin_manager.py

```
@@ -0,0 +1,442 @@
+        module_name = f"plugin_{plugin_file.stem}"
+        
+        # Load module
+        spec = importlib.util.spec_from_file_location(module_name, plugin_file)
+        if not spec or not spec.loader:
+            return None
+        
+        module = importlib.util.module_from_spec(spec)
+        spec.loader.exec_module(module)
+        
+        # Find plugin classes
+        for name, obj in inspect.getmembers(module):
```

Ruff (0.12.2)

Loop control variable `name` not used within loop body

Rename unused `name` to `_name`

(B007)